src/chigame/users/migrations/0002_group_userprofile_groupinvitation_friendinvitation.py

```diff
@@ -0,0 +1,97 @@
+# Generated by Django 4.2.4 on 2023-10-30 16:54
```

Comment:
Both of your migration files are duplicates of files already on dev and cause issues with django migrations when merged with dev. 

Remove them from this PR and conflicts will go away. 

If you go to the github actions corresponding to this PR, you can see the tests fail, they fail because of there being multiple versions of the same migration. Which it asks you do fix with `python manage.py makemigrations --merge`
which will fail unless you actually delete these files / never include them in this commit meant for dev.

Reply:
I just deleted migration files